site_root/input/resources/examples/Observation-vs-001-no-phi.json

```diff
@@ -30,14 +30,23 @@
     "coding": [
       {
         "system": "http://snomed.info/sct",
-        "code": "438949009",
-        "display": "Alive (finding)"
+        "code": "263493007",
+        "display": "Clinical status"
```

Comment:
Should it be `Clinical Status`?

Reply:
@znatty22, the one with a lowercase "s" is actually what is indexed in SNOMED CT: https://browser.ihtsdotools.org/?perspective=full&conceptId1=263493007&edition=MAIN/2020-07-31&release=&languages=en